welcome/cloud-experts-rosa-hcp-sts-explained.adoc

```diff
@@ -0,0 +1,116 @@
+
+STS roles and policies must be created for each {hcp-title} cluster. The ROSA command line interface (CLI) (`rosa`) manages the STS roles and policies. The CLI provides the commands and files to create the roles and policies and an option to allow the CLI to automatically create the roles and policies. 
+//See [insert new xref when we have one for HCP] for more information about the different `--mode` options.
+
+[id="hcp-sts-security"]
+== AWS STS security
+Security features for AWS STS include:
+
+* An explicit and limited set of roles and policies that the user creates ahead of time. The user knows every requested permission and every role used.
+* The service cannot do anything outside of those permissions.
+* There is no need to rotate or revoke credentials. Whenever the service needs to perform an action, it obtains credentials that expire in one hour or less. Credential expiration reduces the risks of credentials leaking and being reused.
+
+{hcp-title} grants users least-privilege permissions with short-term security credentials to specific and segregated IAM roles. The credentials are associated with IAM roles specific to each component and cluster that makes AWS API calls. This method aligns with principles of least-privilege and secure practices in cloud service resource management.
+
+[id="components-specific-to-rosa-hcp-with-sts"]
+== Components of {hcp-title}
+* *AWS infrastructure* - The infrastructure required for the cluster including the EC2 instances, storage, and networking components. See xref:../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-aws-compute-types_rosa-service-definition[AWS compute types] to see the supported instance types for compute nodes and xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-ec2-instances_rosa-sts-aws-prereqs[provisioned AWS infrastructure] for more information on control plane and infrastructure node configuration.
+* *AWS STS* - A method for granting short-term, dynamic tokens to provide users the necessary permissions to temporarily interact with your AWS account resources.
+* *OpenID Connect (OIDC)* - A mechanism for cluster Operators to authenticate with AWS, assume the cluster roles through a trust policy, and obtain temporary credentials from STS to make the required API calls.
+* *Roles and policies* - The roles and policies used by {hcp-title} can be divided into AWS-managed account-wide roles and policies and Operator roles and policies.
++
+The policies determine the allowed actions for each of the roles. See xref:../rosa_architecture/rosa-sts-about-iam-resources.adoc#rosa-sts-about-iam-resources[About IAM resources for ROSA clusters that use STS] for more details about the individual roles and policies.
++
+** The AWS-managed account-wide roles are:
```

Comment:
The account-roles are not AWS-managed, they are created and managed by customers via rosacli tool.